### PR TITLE
Mobile Release v1.104.0

### PR DESCRIPTION
**Pull request by fluiddot**
## Description
Release 1.104.0 of the react-native-editor and Gutenberg-Mobile.

For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/6204

### CODE REVIEW

**Review by dcalhoun — APPROVED** [review]

File: packages/edit-site/lib/lib-font.browser.js
Comment (fluiddot) [inline packages/edit-site/lib/lib-font.browser.js:1]: This file as well as `packages/edit-site/lib/unbrotli.js`, `platform-docs/docs/basic-concepts/_category_.json`, and `platform-docs/package-lock.json` were automatically modified when updating this PR with `trunk`. The `pre-commit` Git hook executed the command `npx lint-staged` and it identified that those files needed code formatting. 

Seems that [this PR](https://github.com/WordPress/gutenberg/pull/53884) introduced the first two files (`packages/edit-site/lib/lib-font.browser.js` and `packages/edit-site/lib/unbrotli.js`), so I'd like to ping the author @matiasbenedetto for a confirmation that the changes are safe to merge into `trunk`.

Regarding the last two (`platform-docs/docs/basic-concepts/_category_.json` and `platform-docs/package-lock.json`), the formatting change seems legit. Nevertheless, I also would like to ping the author @youknowriad of [the PR](https://github.com/WordPress/gutenberg/pull/54375) where it was introduced for a double-check.

Thanks for the help 🙇 !

Comment (youknowriad) [reply packages/edit-site/lib/lib-font.browser.js:1]: Platform-docs changes are all good for me thanks :)

Comment (matiasbenedetto) [reply packages/edit-site/lib/lib-font.browser.js:1]: :wave: Yes, `packages/edit-site/lib/lib-font.browser.js` and `packages/edit-site/lib/unbrotli.js` are safe to merge. These files are part of the `lib-font` package. Soon, [the way of bundling them will change](https://github.com/WordPress/gutenberg/issues/53653) but for now they are OK.

Comment (fluiddot) [reply packages/edit-site/lib/lib-font.browser.js:1]: Great, thank you both Riad and Matias for the feedback 🙇 . I'll proceed with merging the PR then.

Comment (sirreal) [reply packages/edit-site/lib/lib-font.browser.js:1]: I came across these files being autoformatted again. Let's revert the formatting changes introduced here and exclude this directory from autoformatting. From what I understand these aren't files that are authored in Gutenberg: https://github.com/WordPress/gutenberg/pull/62573

**Pull request merged**